pkg/master/master_test.go

```diff
@@ -526,7 +526,7 @@ func testInstallThirdPartyAPIListVersion(t *testing.T, version string) {
 			setupEtcdList(fakeClient, "/ThirdPartyResourceData/company.com/foos/default", test.items)
 		}
 
-		resp, err := http.Get(server.URL + "/apis/company.com/" + version + "/namespaces/default/foos")
+		resp, err := http.Get(server.URL + "/apis/company.com/group/" + version + "/namespaces/default/foos")
```

Comment:
Concrete proposal, since nobody else made one:

```
metadata:
  name: cron-tab.example.com
apiVersion: experimental/v1alpha1
kind: ThirdPartyResource
description: "A specification of a Pod to run on a cron style schedule"
versions:
  - name: v1
  - name: v2experimental1

/thirdparty/example.com/v1/namespaces/<namespace>/crontabs/...
/thirdparty/example.com/v2experimental1/namespaces/<namespace>/crontabs/...
```
thirdparty is the prefix, example.com is the group, v1 and v2experimental1 are the versions, CronTab is the kind.